src/mesh/BoundingBox.hpp

```diff
@@ -80,6 +80,15 @@ class BoundingBox {
   /// the max corner of the bounding box
   Eigen::VectorXd maxCorner() const;
 
+  /// returns the edge length in a specific direction
+  double getEdgeLength(int direction) const;
+
+  /// returns the maximum length of the bounding box in a particular direction
+  double getMaximumEdgeLength() const;
+
+  /// returns the coordinates of the bounding box in a specific direction
+  std::pair<double, double> getDirectionsCoordinates(int direction) const;
```

Comment:
Sure, given that it is an axis-aligned bounding-box.

I suggest making this easier to use though.

```cpp
struct MinMax { double min, double max; };

MinMax operator[](int axis) const;

// allows to write bb[1].min bb[1].max instead of the terrible first and second.

// Alternative

MinMax axis(int axis) const;

// allows to write bb.axis(1).min bb(1).max
```

Reply:
Removed the function